README.md

```diff
@@ -120,6 +120,91 @@ Dane będa przesyłane jako strumień i przetwarzane zgodnie z typem wiadomości
 - Dodaj regulator
 - Dodaj urządzenie
 
+## API
+```text
+// undefined oznacza, że może nie być tego w requeście
+{
+  //GET /devices - zwrotka
+  "devices": [
+    {
+      "id": "string",
+      "regulatorId": "string",
+      "name": "string",
+      "status": "ENUM: ACTIVE | INACTIVE | INVALID | CONNECTING",
+      "type": "ENUM: TEMPERATURE | LIGHT",
+      "data": "string | boolean",
+      "targetData": "string | boolean"
+    }
```

Comment:
To ma być obiekt json z jednym atrybutem `"devices"`, kótry jest tablicą innych obiektów?
Czy nie może być od razu sama tablica? Czy to jest jakaś zalecana metoda, żeby robić tak jak to napisałeś?
Ogólnie to samo mam do wszystkich requestów niżej

Reply:
po prostu nie wiedziałem, czy w springu da się sama tablicę wysłać, ale teraz jak patrzę to się chyba da. Ogólnie u nas w projekcie (allegro) nie widziałem, żeby kiedykolwiek sama tablicę się wysyłało, więc tak założyłem. Jak chcesz mogę zmienić w zależności co będzie ci łatwiej zaimplementować bo dla mnie to 2 min